File: fc-chat/demo/part3/ch3/04-message-system-add-channel/src/main/java/net/prostars/messagesystem/dto/websocket/inbound/FetchUserInvitecodeRequest.java
package net.prostars.messagesystem.dto.websocket.inbound;

import com.fasterxml.jackson.annotation.JsonCreator;
import net.prostars.messagesystem.constant.MessageType;

public class FetchUserInvitecodeRequest extends BaseRequest {

  @JsonCreator
  public FetchUserInvitecodeRequest() {
    super(MessageType.FETCH_USER_INVITECODE_REQUEST);
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/inbound/FetchUserInvitecodeRequest.kt
package com.yscorp.withpush.messagesystem.dto.websocket.inbound

import com.yscorp.withpush.messagesystem.constant.MessageType

class FetchUserInvitecodeRequest @com.fasterxml.jackson.annotation.JsonCreator constructor() :
    BaseRequest(MessageType.FETCH_USER_INVITECODE_REQUEST)
